### ISSUE TITLE
Accessibility fixes: Sitewide

### ISSUE THREAD

**Issue by mariagould** [CDLUC3/ezid#542]
The following accessibility improvements were identified across the EZID site. 

- [x] Make the X for closing show up in all browsers (Doesn't show in Chrome.)
- [x] Add alt text to login “close” image 

- [x] Update alt text and title wherever the help icon (question mark) is used (currently it says "click"). 

<img width="369" alt="Screenshot 2024-01-09 at 08 16 04" src="https://github.com/CDLUC3/ezid/assets/45110557/b7809cf6-e3bf-4d31-bccc-a80328f0e32f">

- [x] Enable the main navigation menu options (Search, Learn, Contact, Login) to be read by a screen reader

- [x] Skip to content link is invisible on focus. Difficult to see focus on selected radio button.
- [x] Find and fix all orphaned form labels across site (Contact Us, My Account, potentially more)
  - Scott's note.  I do not see any labels that don't match to form elements and it's fairly easy to test by clicking on labels since correctly labeled ones automatically move the cursor into the form element when clicked on.  However I found two fields in the advanced form that had duplicate IDs on the page and didn't work correctly because of that (IDs shouldn't be duplicated in a page). I fixed publisher and version IDs.  Perhaps this has already been addressed.  I tried all the forms on the site that I could find.
- [x] Fix contrast errors across site
  - [x] Global nav: Highlighted page title in white. 
  - [x] Heading bars where there is white text on blue background.
  - [x] “Welcome [user]!” message upon login
  - [x] “Logout” button – white text on orange background.
  - [x] Button text – white text on blue background.
  - [x] All non-black text in footer.
  - [x] Breadcrumb showing current page title – gray text on white background.
  - [x] Code samples on info pages.
  - [x] Headings on info pages.

**Comment by adambuttrick** [comment]
Included in https://github.com/CDLUC3/ezid/releases/tag/v3.3.2.
